src/app/cube/browse/browse.component.ts

```diff
@@ -37,6 +37,7 @@ export class BrowseComponent implements AfterViewInit, OnDestroy {
     sortType: undefined,
     collection: '',
     fileTypes: [],
+    status: [LearningObject.Status.RELEASED]
```

Comment:
This still works even without a text query? (I.e. just going to the browse page from the homepage with no query)

Reply:
I was able to confirm that it does correctly load only released learning objects when there is no text query